tools/LibuvCopier/project.json

```diff
@@ -4,19 +4,16 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "Newtonsoft.Json": "8.0.3"
+    "Newtonsoft.Json": "9.0.1"
   },
   "frameworks": {
-    "netcoreapp1.0": {
+    "netcoreapp1.1": {
       "dependencies": {
         "Microsoft.NETCore.App": {
-          "version": "1.0.0-rc2-3002702",
+          "version": "1.1.0-*",
```

Comment:
Could we pin these to the LTS builds (1.0.0)? That way we can remove the reference to the aspnetcore feed from NuGet.config.